### ISSUE TITLE
Validation Loss Exploding to 10^5

### ISSUE THREAD

**Issue by lsaeuro** [Lightning-AI/lightning#14085]
## 🐛 Bug

Validation loss  is exploding and I figured out that the problem is already in the forward method of my model.
During Validation , the input tensors after a batch norm or an activation function, does not change a lot, while during training it is rescaled.
![image](https://user-images.githubusercontent.com/107695630/183413269-30f59b7e-e2dc-46cd-aabf-71579bbe54e6.png)

I found something similar here [in the Lightning Forum](https://forums.pytorchlightning.ai/t/my-training-loss-and-validation-loss-are-correct-but-my-validation-loss-is-exploding/1650). 
Are there some particular modes in the validation loss that I should turn on or off?

### Environment

* CUDA:
	- GPU:
		- NVIDIA RTX A6000
	- available:         True
	- version:           10.2
* Packages:
	- numpy:             1.23.0
	- pyTorch_debug:     False
	- pyTorch_version:   1.11.0+cu102
	- pytorch-lightning: 1.7.0
	- tqdm:              4.64.0
* System:
	- OS:                Linux
	- architecture:
		- 64bit
		- ELF
	- processor:         x86_64
	- python:            3.8.13
	- version:           #44~20.04.1-Ubuntu SMP Fri Jun 24 13:27:29 UTC 2022

**Comment by lsaeuro** [comment]
Thank you @awaelchli , indeed this  was the problem : 

> > During Validation , the input tensors after a batch norm or an activation function, does not change a lot, while during training it is rescaled.

I solved it setting in the BatchNorm2D track_running_stats = False , but actually I don't know if this is a wrong work around, I will also try to do what you suggested. Thank you!

**Comment by awaelchli** [comment]
Let me know if you have further questions regarding this. Instead of setting `track_running_stats = False`, alternatively you could also try to tune the momentum parameter on the batchnorm layer.
